paddle/operators/send_op.cc

```diff
@@ -37,6 +37,7 @@ class SendOp : public framework::OperatorBase {
     auto ins = Inputs("X");
     auto outs = Outputs("Out");
     std::vector<std::string> epmap = Attr<std::vector<std::string>>("epmap");
+    bool do_get = Attr<bool>("DoGet");
```

Comment:
Can we infer `do_get` from `outs.size() > 0`? So we don't need an additional attribute.

Reply:
That's a great idea! Thanks.